frontend/src/components/managePermissions/table/TablePlaceholder.vue

```diff
@@ -2,7 +2,7 @@
 
 <template>
     <div class="no-app-selected">
-        <IconCapitol />
+        <IconCapitol role="presentation" title="Icon Capitol" />
```

Comment:
Added a title for WCAG guidelines but in essence this image has no useful meaning to the user, so it was set to presentation as well.